packages/fe-fpm-writer/templates/building-block/field/View.xml

```diff
@@ -0,0 +1,8 @@
+<<%- macrosNamespace %>:Chart
+    id="<%- data.id %>"<% if (data.metaPath) { %>
```

Comment:
Probably Field rather than Chart :)

Reply:
Thanks for catching it. :)

Reply:
Updated the template and test snapshots.